src/app/profiles/profiles.default.ts

```diff
@@ -0,0 +1,116 @@
+// --------- BEGIN RUNBOX LICENSE ---------
+// Copyright (C) 2016-2018 Runbox Solutions AS (runbox.com).
+//
+// This file is part of Runbox 7.
+//
+// Runbox 7 is free software: You can redistribute it and/or modify it
+// under the terms of the GNU General Public License as published by the
+// Free Software Foundation, either version 3 of the License, or (at your
+// option) any later version.
+//
+// Runbox 7 is distributed in the hope that it will be useful, but
+// WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+// General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Runbox 7. If not, see <https://www.gnu.org/licenses/>.
+// ---------- END RUNBOX LICENSE ----------
+
+import { Component } from '@angular/core';
+import { RMM } from '../rmm';
+import { RunboxWebmailAPI } from '../rmmapi/rbwebmail';
+import { timeout, share, retry } from 'rxjs/operators';
+import { MatSnackBar } from '@angular/material/snack-bar';
+
+@Component({
+    selector: 'app-profiles-default',
+    styleUrls: ['profiles.default.scss'],
+    templateUrl: 'profiles.default.html',
+})
+export class DefaultProfileComponent {
+    field_errors: any;
+    profiles: any[] = new Array();
+    selected: any;
+
+    constructor(public rmm: RMM, public rmmapi: RunboxWebmailAPI, private snackBar: MatSnackBar) {
+        this.selectCurrentDefault();
+    }
+
+    async fetchProfiles() {
+        const froms = await this.rmmapi.getFromAddress().toPromise();
+        // Sort emails alphabetically
+        this.profiles = froms.sort((a, b) => (a.email < b.email ? -1 : 1));
+    }
+
+    async selectCurrentDefault() {
+        await this.fetchProfiles();
+        const defaultProfiles = [];
+        for (const profile of this.profiles) {
+            if (profile.priority === '0' || profile.priority === 0) {
+                defaultProfiles.push(profile);
+            }
+        }
+        if (defaultProfiles.length === 1) {
+            this.selected = defaultProfiles[0];
+        } else {
+            for (const profile of defaultProfiles) {
```

Comment:
Done.
I believe there is no way to have more `main` accounts anymore. I already talked about this with @hernan604 because there was a possibility to have multiple before. He fixed it and now it's all pretty and shiny.